frameworks/pytorch/get_started_mnist_deploy.ipynb

```diff
@@ -6,23 +6,15 @@
    "source": [
```

Comment:
<p>Grammar:</p><p> </p><p><code>before put it into the production</code> -&gt; <code>before putting it into production</code>   </p><p> </p><p> <code>what they suppose to be, then the endpoint point is going to work</code> -&gt; <code>what they are supposed to be, then the endpoint is going to work</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Done.</p>